### ISSUE TITLE
[Feature Request] Screenshot Helper Granularity Options

### ISSUE THREAD

**Issue by KazyEXE** [vrcx-team/VRCX#744]
I love Screenshot Helper as it helps me figure out what world I was in when I took a photo, but I hate that it also is sort of privacy invasive as it exposes everyone who was in the instance at the time. I'd love to be able to keep the World ID data but not the players in the instance.

**Comment by GroovyTeacup** [comment]
True. I feel like I thought of this when I first implemented it but then forgot about it. I'll prob add options for it soon
